### ISSUE TITLE
Error loading shared library libudev.so.1: No such file or directory

### ISSUE THREAD

**Issue by aronand** [aronand/godot-alpine#1]
When trying to start the editor, the user will get the following error:

```
Error loading shared library libudev.so.1: No such file or directory
```

Looking at `platform/linuxbsd/detect.py` in Godot's repo, it seems that it will try to use `libudev` if the host that built the binary had it available at the time:

```python
if platform.system() == "Linux":
    env.Append(CPPDEFINES=["JOYDEV_ENABLED"])
    if env["udev"]:
        if not env["use_sowrap"]:
            if os.system("pkg-config --exists libudev") == 0:  # 0 means found
                env.ParseConfig("pkg-config libudev --cflags --libs")
                env.Append(CPPDEFINES=["UDEV_ENABLED"])
            else:
                print_warning("libudev development libraries not found. Disabling controller hotplugging support.")
                env["udev"] = False
        else:
            env.Append(CPPDEFINES=["UDEV_ENABLED"])
else:
    env["udev"] = False  # Linux specific
```

To fix this issue we either have to get `libudev` into the image, or remove it from the builder (as it seems, `ubuntu-latest` has this library by default).

**Comment by aronand** [comment]
Verified that the editor built in my local Alpine container works doesn't run into this problem (as expected).

**Comment by aronand** [comment]
Setting `udev=no` in scons fixed this particular issue, though now the editor crashes. Making one more build with `execinfo=no`, as the build logs mention this in conjunction with distributions that don't have `glibc` (e.g. Alpine).

If that doesn't work, or even if it does, it might be worth looking into [running the job in a container](https://docs.github.com/en/actions/writing-workflows/choosing-where-your-workflow-runs/running-jobs-in-a-container).

**Comment by aronand** [comment]
This issue is solved, however a new one (#3) raises its head.
